src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationManagementController.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.hmcts.reform.pip.data.management.controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.pip.data.management.models.PublicationFileSizes;
+import uk.gov.hmcts.reform.pip.data.management.service.PublicationManagementService;
+import uk.gov.hmcts.reform.pip.model.authentication.roles.IsAdmin;
+import uk.gov.hmcts.reform.pip.model.publication.FileType;
+
+import java.util.UUID;
+
+@Slf4j
+@RestController
+@Tag(name = "Controller to handle the retrieval of publication files and file' properties, and generation of "
```

Comment:
I'm wondering about the naming on this file vs Publication Controller. It's hard on first glance to know the split between them.

I wonder if this should be renamed to be something more closely aligned with the extra file formats it gives?

Reply:
Updated the description within the tag annotation